mkdocs_drawio_exporter/exporter.py

```diff
@@ -248,6 +248,34 @@ def validate_config(self, config: Configuration):
                     'embed_format', config['embed_format'],
                     'cannot inline content of non-SVG format')
 
+    def set_css_classes(self, svg_content, prefix="--drawio-color"):
+        """Rewrite SVG styles to use CSS variables.
+        Will replace all `stroke` and `fill` attributes with CSS variables
+        such as `var(--drawio-color-ffffff)`.
+
+        :param str svg_content: SVG content.
+        :param str prefix: CSS variable prefix.
+
+        :return str: SVG content with styles rewritten.
+        """
+        pattern = re.compile(
+            (r'(fill|stroke)\s*='
+             r'\s*\"(\#[A-Fa-f0-9]+|'
+             r'rgb\([0-9]{1,3}\s*,\s*[0-9]{1,3}\s*,\s*[0-9]{1,3}\))\"'))
```

Comment:
We'll need tests for this regex. Are you happy to add these, or would you rather I took a look?

Reply:
Yes I can add some tests